src/components/LibraryItem.tsx

```diff
@@ -179,6 +179,7 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
     }
 
     onImageLoadFail(event: any) {
+        event.target.orgSrc = event.target.src;
```

Comment:
this lets us lookup what the originalSrc url was for an icon, even if we have fallenback to the default dynamo icon.